Packs/DemistoRESTAPI/Scripts/GetIncidentTasks/GetIncidentTasks.py

```diff
@@ -0,0 +1,147 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+from typing import Any
+
+
+TASK_STATES = {
+    'new': '',
+    'inprogress': 'inprogress',
+    'completed': 'Completed',
+    'waiting': 'Waiting',
+    'error': 'Error',
+    'looperror': 'LoopError',
+    'skipped': 'WillNotBeExecuted',
+    'blocked': 'Blocked'
+}
+
+
+''' STANDALONE FUNCTION '''
+
+
+def get_playbook_tasks(tasks: list) -> list:
+    """Get the tasks of a playbook recursively
+
+    Args:
+        tasks (list): the tasks of the playbook
+
+    Returns:
+        list: the tasks fo the playbook including all sub-playbook tasks
+    """
+    if not tasks:
+        return []
+
+    ready_tasks = []
+    for task in tasks:
+        if task.get('type') == 'playbook' and task.get('subPlaybook'):
+            sub_playbook_tasks = task.get("subPlaybook", {}).get("tasks", {}).values()
+            ready_tasks.extend(get_playbook_tasks(list(sub_playbook_tasks)))
+        ready_tasks.append(task)
+
+    return ready_tasks
+
+
+def get_states(states: list[str]) -> list[str]:
+    """Get the internal states names for the given search states
+
+    Args:
+        states (list[str]): a list of states to search
+
+    Returns:
+        list[str]: a list of the internal names of the states given in the input or all possible states, if input was empty
+    """
+    if 'error' in states:
```

Comment:
@shmuel44 Shall we leave this in here, or remove it, since we would have the LoopError as a dedicated possible stat

Reply:
We have decided to keep it the same as it was in the old script.